.github/workflows/ci.yml

```diff
@@ -118,6 +121,74 @@ jobs:
         run: |
           ddev composer install
           ddev pull platform -y
+
+      - name: Run updates and Validate Config
+        id: validate_config
+        run: |
+          # rebuild cache so that config cache is not absolete
+          ddev drush cache:rebuild -y
+
+          # identify set of configuration changes before running updatedb and store it in a text file.
+          # awk NF is used to filter blank lines.
+          ddev drush config:status --fields name --format list | awk NF > ${{ runner.temp }}/initial_config.txt
+
+          # run updates and build cache
+          ddev drush updatedb --yes
+          ddev drush cache:rebuild --yes
+
+          # identify set of configuration changes after running updatedb and store it in a text file.
+          ddev drush config:status --fields name --format list | awk NF > ${{ runner.temp }}/updated_config.txt
+
+          # sort the lines within files as comm requires sorted files
+          sort ${{ runner.temp }}/initial_config.txt -o ${{ runner.temp }}/sorted_initial_config.txt
+          sort ${{ runner.temp }}/updated_config.txt -o ${{ runner.temp }}/sorted_updated_config.txt
```

Comment:
Instead of a separate `sort`, let's pipe `awk` output into `sort`.

Reply:
I know that `sort` needs to do additional work in a pipe but it's okay. In our case, it is not likely to get big. I'd rather remove extra lines here. That said, I don't feel strongly about this. It is fine the way it is right now as well.

Reply:
@hussainweb I am removing this. I noticed `config:status` already produces sorted result. I confirmed checking drush code and there is a ksort method used. 

https://github.com/drush-ops/drush/blob/151b70439cdacda5b95546ca853f271d0fe981cf/src/Commands/config/ConfigCommands.php#L295